linchpin/provision/roles/libvirt/tasks/provision_libvirt_node.yml

```diff
@@ -27,8 +27,7 @@
 
 - name: "check cloud_config is used or not"
   set_fact:
-    cloud_config_used: false
-  when: cloud_config['users'] == [] and cloud_config['run_commands'] == []
+    cloud_config_used: "{{ true if (cloud_config['users'] }} == [] and cloud_config['run_commands'] == []) else false }}"
```

Comment:
You have two `}}` and only one `{{` .. I know sk already merged, but this does not work for me